src/w3c/templates/headers.js

```diff
@@ -22,6 +22,20 @@ function getSpecTitleElem(conf) {
     specTitleElem.textContent = conf.title;
     specTitleElem.id = "title";
   }
+  if (conf.isPreview && conf.prNumber) {
+    if (conf.prUrl)
+      specTitleElem.prepend(
```

Comment:
you could do:
```JS
 const {childNodes}  = html`Preview of PR <a href=${conf.prUrl}>#${conf.prNumber}</a>:`;
 specTitleElem.prepend(...childNodes);
```
And that gets rid of the span.

Reply:
Can I use innerHTML instead? ESLint causes the the string to be split across lines, adding whitespace, which I cannot trim.

Reply:
we prefer not to use `innerHTML` because hyperHTML gives us some security assurances. If you need to disable ESLint for a particular line, then that's acceptable. Just put a comment.

Reply:
A comment asking not to remove the comment disabling eslint prettier?